capa/main.py

```diff
@@ -123,6 +124,23 @@
 E_INVALID_INPUT_FORMAT = 25
 E_INVALID_FEATURE_EXTRACTOR = 26
 
+root_logger = logging.getLogger()
+logging.basicConfig(level=logging.NOTSET)
```

Comment:
you have the right idea in the commented-out code in `handle_common_args` below: configure the logging module within a library function, not within code that might be imported by another program (like these top level statements).

Reply:
im going to continue my review on the commented-out code.